### PR TITLE
BBC

### PR DESCRIPTION
**Pull request by akesterton**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- ...
- ...
- ...

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

File: knowledge/studios/bbc_studios/qna.txt
Comment (jjasghar) [inline knowledge/studios/bbc_studios/qna.txt:1]: Please change this name to `qna.yaml`.

File: knowledge/studios/bbc_studios/qna.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+created_by:
```
Comment (jjasghar) [inline knowledge/studios/bbc_studios/qna.txt:1]: Please add your GitHub username here.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please make these changes then we can re-run the linter.

File: knowledge/studios/bbc_studios/qna.txt
Comment (akesterton) [reply knowledge/studios/bbc_studios/qna.txt:1]: Apologies, misread the instructions.  Changed as requested.

File: knowledge/studios/bbc_studios/qna.txt (hunk 1 above)
Comment (akesterton) [reply knowledge/studios/bbc_studios/qna.txt]: And missed this bit too - changed as requested

File: knowledge/studios/bbc_studios/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+created_by:
```
Comment (jjasghar) [inline knowledge/studios/bbc_studios/qna.yaml:1]: Please add your GitHub Username here

File: knowledge/studios/bbc_studios/attribution.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Title of work: BBC Studios
+Link to work: https://en.wikipedia.org/wiki/BBC_Studios
+Revision: https://en.wikipedia.org/w/index.php?title=BBC_Studios&oldid=1221919574
+License of the work: CC-BY-SA-4.0
+Creator names: akesterton
```
Comment (jjasghar) [inline knowledge/studios/bbc_studios/attribution.txt:5]: This should we Wikipedia Authors

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix these issues.

**Comment by jjasghar** [comment]
Please fix the linting issues.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the linting issues.

**Comment by jjasghar** [comment]
Closing because of combined PR here: https://github.com/instructlab/taxonomy/pull/1187

**Pull request closed without merging**